src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/SuppressMessage.cs

```diff
@@ -81,5 +81,33 @@ public override bool Equals(object obj)
 
 			return Equals(message);
 		}
-	}
+
+        public int CompareTo(SuppressMessage other)
+        {
+            if (Equals(other))
+            {
+                return 0;
+            }
+
+            var idComparison = string.Compare(Id, other.Id);
```

Comment:
I think that comparison should be ordinal, case-sensitive.
`String.Compare` with two parameters uses the current culture to perform the comparison (see [the docs](https://docs.microsoft.com/en-us/dotnet/api/system.string.compare?view=netframework-4.7.2#System_String_Compare_System_String_System_String_)).

Reply:
@dropsonic I agree